packages/astro/src/cli/dev/index.ts

```diff
@@ -20,6 +20,7 @@ export async function dev({ flags }: DevOptions) {
 					['--host <custom-address>', `Expose on a network IP address at <custom-address>`],
 					['--open', 'Automatically open the app in the browser on server start'],
 					['--force', 'Clear the content layer cache, forcing a full rebuild.'],
+					['--allowed-hosts', 'Specify a comma-separated list of allowed hosts.'],
```

Comment:
Should we precise that `true` is also a valid value? I know, it might be difficult to keep it brief if we add this information... Maybe something like:
```suggestion
					['--allowed-hosts', 'Specify a comma-separated list of allowed hosts or allow any hostname.'],
```